src/state/stateSingle.test.ts

```diff
@@ -406,6 +406,42 @@ describe("stateSingle", () => {
         sub.unsubscribe()
       })
 
+      it("returns a promise if the latest emitted value was SUSPENSE", async () => {
```

Comment:
This test was already working, I just added it to highlight the consistency of this fix.